src/routes/web.php

```diff
@@ -1,18 +1,12 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
-
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
+use App\Http\Controllers\CountryController;
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('index', ['title' => 'Affected Countries']);
```

Comment:
```suggestion
    return view('index')
        ->with('title', 'Affected Countries');
```
Vietoj array naudojant objektinį kreipinį bus geresnis readability.